code/__DEFINES/subsystems.dm

```diff
@@ -46,6 +46,7 @@
 // The numbers just define the ordering, they are meaningless otherwise.
 #define INIT_ORDER_PROFILER	101
 #define INIT_ORDER_TITLE 100 // Load this quickly so people dont see a blank lobby screen
+#define INIT_ORDER_VIS 22
```

Comment:
Should this really be loading this soon? Especially before the GC.

Reply:
Duly noted. I put it where there was an empty spot. Where do you think would be the best spot. After lighting, maybe?